File: Arrays/LargestElement/largestElement.py
class Solution:
    def largest(self, arr):
        # code here
        length = len(arr)
        if length == 1: return arr[0]
        max = arr[0]
        for i in range(1,length):
            if max<arr[i]: max = arr[i]
        return max
        

File: Arrays/LargestElement/largestElement.cpp
#include <vector>
using namespace std;

class Solution {
  public:
    int largest(vector<int> &arr) {
        // code here
        if(arr.size()==1)return arr[0];
        int max = arr[0];
        for(int i=1;i<arr.size();i++){
            if(max<arr[i])max = arr[i];
        }
        return max;
    }
};
